composer.json

```diff
@@ -14,7 +14,7 @@
 	"require": {
 		"php": ">=5.5.0",
 		"benestar/asparagus": "~0.4",
-		"monolog/monolog": "~1.18",
+		"monolog/monolog": "~1.18||^2.0.0",
```

Comment:
Should this be a double pipe? `||`?

Reply:
I think so, yes? https://getcomposer.org/doc/articles/versions.md#version-range

Reply:
interesting, yes the docs seem to suggest the right way is with a double pipe, despite the line below this using single pipes!

Reply:
Heh, I had not looked at the line below :D